framework-plugins/lisk-framework-monitor-plugin/src/schema.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+export const postBlockEventSchema = {
+	id: 'monitor/postBlockEvent',
+	type: 'object',
+	required: ['block'],
+	properties: {
+		block: {
+			type: 'string',
+			format: 'hex',
+		},
+	},
+};
+
+export const transactionAnnouncementSchema = {
+	id: 'monitor/transactionAnnouncement',
+	type: 'object',
+	required: ['transactionIds'],
+	properties: {
+		transactionIds: {
+			type: 'array',
+			items: {
+				type: 'string',
+				format: 'hex',
+			},
+		},
+	},
+};
```

Comment:
This does not match exactly with what we use in the Node. 

https://github.com/LiskHQ/lisk-sdk/blob/4409d34102b5bc6703fbc48813bd1f7557ee4a93/framework/src/node/transport/schemas.ts#L51-L65

Reply:
number of the list doesnt really matter here. it just needs to be hex string array to process further